darwin/future/core/items/move_items_to_folder.py

```diff
@@ -44,7 +44,8 @@ def move_list_of_items_to_folder(
             if isinstance(dataset_ids, list)
             else [dataset_ids],
             **filters,
-        }
+        },
+        "path": path,
```

Comment:
In PY-560 refactoring, I accidentally removed path & priority from their core methods respectively

Added them back